content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md

```diff
@@ -140,11 +141,16 @@ Use the following steps to provision a new AWS cluster:
     | **PreferNoSchedule**| The system will avoid placing a non-tolerant pod to the tainted node but is not guaranteed.
     | **NoExecute**|  New pods will not be scheduled on the node, and existing pods on the node if any on the node will be evicted they do not tolerate the taint. |
 
-14. Click on **Next**.  
-    
-15. The settings page is where you can configure the patching schedule, security scans, backup settings, and set up Role Based Access Control (RBAC). Review the cluster settings and make changes if needed. Click on **Validate**.
 
-16. Review the settings summary and click on **Finish Configuration** to deploy the cluster. Provisioning IaaS clusters can take 15 - 30 minutes depending on the cluster profile and the node pool configuration.
+14. If you checked the **Static Placement** box in the **Cluster config** page, you can specify additional AWS [security groups](https://docs.aws.amazon.com/vpc/latest/userguide/security-groups.html) to apply to the worker group nodes. Use the **Additional Security Groups (Optional)** drop-down Menu to select additional security groups.
```

Comment:
Our style is to boldface **drop-down Menu**. Can this be reworded to get around boldfacing "Additional Security Groups (Optional)"?